cmd/erasure-healing.go

```diff
@@ -560,6 +579,7 @@ func (er erasureObjects) healObject(ctx context.Context, bucket string, object s
 				})
 				if len(inlineBuffers) > 0 && inlineBuffers[i] != nil {
 					partsMetadata[i].Data = inlineBuffers[i].Bytes()
+					partsMetadata[i].SetInlineData()
```

Comment:
This a bug that is fixed to ensure that the partsMetadata contains the right set of headers to indicate after healing that this is an inlined data.